tests/e2e/e2e_setup_test.go

```diff
@@ -185,10 +188,16 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		}
 	}
 
-	if !skipUpgrade {
+	if !skipUpgrade && !isFork {
```

Comment:
once you have more than one `else if`, it's best to convert to a `switch` clause for better readability